projects/demo/src/modules/components/slider/examples/import/insert-template.txt

```diff
@@ -1,6 +1 @@
-<tui-slider [formControl]="control"
-            max="10"></tui-slider>
-
-<tui-slider [formControl]="testValue"
-            [max]="10"></tui-slider>
-
+<input tuiSlider [formControl]="control"/>
```

Comment:
what do you think about using markdown? as it is in the example with the avatar demo